expand.py

```diff
@@ -678,6 +688,19 @@ def expand(cfg, lines, f, isfirst=False):
                     except:
                         cnt = "0"
                     f.write(cnt)
+                elif kw == "ASSIGN":
+                    args = argm.group(1).split(",")
+                    output = StringIO.StringIO()
+                    expand(cfg, [args[1]], output, isfirst)
+                    value = output.getvalue()
+                    output.close()
+                    try:
+                        v = cfg.eval_expr(value)
+                    except:
+                        v = 0
```

Comment:
Is this a useful default in some cases? If so, great, if not, then maybe this really should raise an exception and let the user know that they misused the ASSIGN directive.

Reply:
It probably isn't useful.  There are a lot of other instances of this type of behavior...